.github/workflows/master-push.yml

```diff
@@ -0,0 +1,87 @@
+name: Push to master
```

Comment:
I understand these is the same as for PR pushes, without the linter and coverage? I was thinking if they can be unified, but maybe that would be less readable...

Reply:
It is very similar, and there may be a way to unify, but I don't know if one can apply `if` clauses to the matrix definition - I've never seen any like that. You can certainly test what branch you are and so on, so if we could define two different sets of matrices that would be the way to go.

> was thinking if they can be unified, but maybe that would be less readable

Yes, I felt that would really clutter it up.  

I'm hoping once these are in place they will very rarely be edited, but it is a valid point that we have a lot of duplication between the two.  I tried to find some way to incorporate one workflow into another, but any docs or discussion I saw online (dated up to a few months ago) said it was not possible.

Reply:
Ok, I didn't delve into it. Looks like this is the best way to do it in this case. Thanks!